wordpress-mobile/WordPress-Android
# Issue/8643 full screen dialog

### Fix
Add a full-screen dialog interface for the ***Customize Your Site*** and ***Grow Your Audience*** tasks as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8643.  See the screenshots below for illustration.

![v2_list](https://user-images.githubusercontent.com/3827611/49414702-f4611600-f739-11e8-8e90-695079c71c3b.png)

#### Note
The completed tasks are shown at the bottom of the list without the expandable section as shown in the design mockups.  That will be addressed in a separate pull request as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8645.  There are three `// TODO: Quick Start -` items included in these changes that are related to analytics and new tasks.  They will be addressed in a separate pull requests as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8647 and https://github.com/wordpress-mobile/WordPress-Android/issues/8649.  The old seven tasks excluding ***Create your site*** will start the task tutorial, but the new four tasks will not.  The old seven task are listed below:
- Create your site
- View your site
- Browse themes
- Customize your site
- Add social media account
- Publish a post
- Follow other sites

### Test
0. Select site with Quick Start in progress.
1. Go to ***Sites*** tab.
2. Notice ***Next Steps*** view is shown.
3. Tap ***Customize Your Site*** type.
4. Notice ***Customize Your Site*** tasks are shown.
5. Tap old task in list.
6. Notice task tutorial is shown.
7. Tap ***Customize Your Site*** type.
8. Tap new task in list.
9. Notice task tutorial is not shown.
10. Tap ***Grow Your Audience*** type.
11. Notice ***Grow Your Audience*** tasks are shown.
12. Tap old task in list.
13. Notice task tutorial is shown.
14. Tap ***Grow Your Audience*** type.
15. Tap new task in list.
16. Notice task tutorial is not shown.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.
